variables.tf

```diff
@@ -51,7 +51,7 @@ variable "name" {
 
 variable "backends" {
   description = "List of backends, should be a map of key-value pairs for each backend, must have the 'group' key."
-  type        = list(object({ group = string, description = string }))
+  type        = list(object({ group = string, description = string, failover = bool }))
```

Comment:
Can we change this type to `list(any)`, so that we don't break backwards-compatibility.

Reply:
@morgante if you want we can do that, but that would be at the expense of loosing type safety. 
Personally I'd rather have it like this, and have the type safety in your IDE and get errors during terraform plan rather than wait for apply to get an error back. But if you want I'll move it to list(any) right now?